tests/e2e/cypress/fixtures/eligibility.json

```diff
@@ -0,0 +1,14 @@
+{
+  "eligible": {
```

Comment:
@thekaveman Any feedback on how I've organized this json file and how I'm using it?